spyder/app/tour.py

```diff
@@ -1164,6 +1209,26 @@ def set_tour(self, index, frames, spy_window):
 
     def start_tour(self):
         """ """
+        self.spy_window.setUpdatesEnabled(False)
+        if self.spy_window.isFullScreen() or self.spy_window.fullscreen_flag:
+            if sys.platform == 'darwin':
+                self.spy_window.setUpdatesEnabled(True)
+                msg_title = _("Request")
+                msg = _("To run the tour, please press the green button in the top left corner of "
+                        "the screen to take Spyder out of fullscreen mode.")

```

Comment:
In conventional English usage, it would be "round green button".

However, I'm not sure how "round" makes it any clearer that it is not part of the Spyder interface, and Spyder doesn't have any green buttons, round or otherwise, in that area of the default interface.

If you do want to be more explicit on that point. we could say something like "please press the green button on the left of the Spyder window's title bar, visible when moving your mouse to the top of the screen." but I'm not sure its necessary or really makes it clearer what to do as opposed to just complicating things further. I don't think don't need to really need to know whether its part of the Spyder interface, just where to find it.

Reply:
> Spyder doesn't have any green buttons, round or otherwise, in that area of the default interface.

Sure there is: the Run button. That's why I wanted to clarify what green button we're talking about.

> If you do want to be more explicit on that point. we could say something like "please press the green button on the left of the Spyder window's title bar, visible when moving your mouse to the top of the screen."

I like your suggestion until "title bar", so I changed mine to use yours instead.

Reply:
> Sure there is: the Run button. That's why I wanted to clarify what green button we're talking about.

That's in the top center for me, but its conceivable it could be different at different screen widths or if people re-arrange their toolbars.

> I like your suggestion until "title bar", so I changed mine for yours instead.

Sounds good to me :+1: